src/components/TabComponent/TabComponent.js

```diff
@@ -21,11 +22,9 @@ export default class TabComponent extends PureComponent {
                         <div
                             key={index}
                             onClick={() => this.onTabChange(index)}
-                            className={
-                                `${  index === this.state.activeTab
-                                    ? 'tab active'
-                                    : 'tab'}`
-                            }
+                            className={classnames('tab', {
+                                'active': index === this.state.activeTab
```

Comment:
if className write in 1 word you can write it without quotes

Reply:
okay